src/main/java/frc/robot/commands/Drivetrain/AutoCommand.java

```diff
@@ -0,0 +1,204 @@
+package frc.robot.commands.Drivetrain;
+
+import com.pathplanner.lib.PathPlanner;
+import com.pathplanner.lib.PathPlannerTrajectory;
+import com.pathplanner.lib.commands.PPRamseteCommand;
+import com.pathplanner.lib.PathConstraints;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.Constants;
+import frc.robot.commands.Elevator.Elevator;
+import frc.robot.commands.Elevator.ElevatorDown;
+import frc.robot.commands.Elevator.ElevatorUp;
+import frc.robot.commands.Grabber.Grabber;
+import frc.robot.commands.Grabber.GrabberCommand;
+import frc.robot.commands.Shooter.Shooter;
+import frc.robot.commands.Shooter.ShooterCommand;
+
+import java.nio.file.Path;
+
+public class AutoCommand extends SequentialCommandGroup{
+    public AutoCommand(Drivetrain drivetrain, String autoChoice, Elevator elevator, Grabber grabber, Shooter shooter){
+        addRequirements(drivetrain);
```

Comment:
Should also add elevator and shooter, don't want those manual controls to get in the way